documentation/tutorials/philosophy.md

```diff
@@ -8,8 +8,10 @@ The philosophy behind Ash allows us to build an extremely flexible and powerful
 
 To this end, there are many prebuilt extensions to use, but there is also a rich suite of tools to build your _own_ extensions. In this way, you can make the framework work for you, instead of struggling to fit your application to a strictly proscribed pattern. Use as much of Ash as you can, and leverage the amazing Elixir ecosystem for everything else.
```

Comment:
Change "proscribed" to "prescribed" - https://www.merriam-webster.com/words-at-play/prescribe-vs-proscribe-usage-difference